subaru_outback_2015_eyesight.dbc

```diff
@@ -369,3 +371,6 @@ CM_ SG_ 880 Steering_Voltage_Flat "receives later than 371";
 CM_ SG_ 880 NEW_SIGNAL_1 "0 in 2017";
 CM_ SG_ 880 NEW_SIGNAL_4_2017 "1 in 2017";
 CM_ SG_ 880 NEW_SIGNAL_5_2017 "1 in 2017";
+CM_ SG_ 1745 Units "0 = Metric, 1 = Imperial";
+CM_ SG_ 1745 Units2 "0 = Metric, 1 = Imperial";
```

Comment:
I think values would be more appropriate than a comment here

Reply:
Changed comments to values